pkg/reconciler/ingress/ingress_test.go

```diff
@@ -284,7 +284,7 @@ func TestReconcileTLS(t *testing.T) {
 			rp(secret(secretName, nsName)),
 		},
 		WantUpdates: []clientgotesting.UpdateActionImpl{{
-			Object: gw(defaultListener, tlsListener("secure.example.com", nsName, secretName)),
+			Object: gw(defaultListener, tlsListener("example.com", nsName, secretName)),
```

Comment:
these are needed, as the hosts in `tls` now has to match the one in `KnativeIngress`, which is `example.com`.